met-api/src/met_api/services/cdogs_api_service.py

```diff
@@ -54,6 +54,7 @@ def generate_document(self, template_hash_code: str, data, options):
 
     @staticmethod
     def _post_generate_document(json_request_body, headers, url):
+        # pylint: disable=missing-timeout
         response = requests.post(url, data=json_request_body, headers=headers)
```

Comment:
Was there a reason we couldn't remedy this linter issue? Even if we don't want to specify a timeout value, couldn't we just set it to None?

```suggestion
        response = requests.post(url, data=json_request_body, headers=headers, timeout=None)
```

Reply:
Updated as suggested, thanks.